cmd/btcctl/version.go

```diff
@@ -18,11 +18,11 @@ const semanticAlphabet = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqr
 const (
 	appMajor uint = 0
 	appMinor uint = 22
-	appPatch uint = 0
+	appPatch uint = 1
 
 	// appPreRelease MUST only contain characters from semanticAlphabet
 	// per the semantic versioning spec.
-	appPreRelease = "beta"
+	appPreRelease = ""
```

Comment:
So `btcd` is now _officially_ out of beta? 🤩

Reply:
Something I highly suggest is to separate the *product* version (aka the release versions of btcd) from the *go module* versions.  I know it might be a little annoying, but Go hijacked semver for its modules and, as things like this show, module versions really need to be independent.

The model I use for dcrd that has worked out really well is for each release to create a `release-vX.Y` branch and on that branch update the version info to remove the pre-release details, remove any overrides from `go.mod`, etc.  Then, bump the to the next pre-release version on the master branch where development continues as normal.

That way all releases have their own branch that you can back port to if it is necessary and it is decoupled from the go module version(s).

Reply:
> Something I highly suggest is to separate the product version (aka the release versions of btcd) from the go module versions. 

Yeah I'm slowly coming around to this realization myself. Takes a bit more work, but worth it not to cause headaches for consumers of the modules.